--- a/src/intlconv.c	2018-05-11 18:13:14.827925726 +0200
+++ b/src/intlconv.c	2018-05-11 22:41:26.067916947 +0200
@@ -23,29 +23,11 @@ tvh_iconv(iconv_t cd, char **inbuf, size
   return iconv(cd, inbuf, inbytesleft, outbuf, outbytesleft);
 }
 
-static void
-intlconv_test( void )
-{
-  /* The string is "Yellow Horse" in Czech for the curiosity */
-  const char *charset = intlconv_charset_id("ASCII", 1, 1);
-  char *s = intlconv_utf8safestr(charset, "ŽluťoučkýKůň", 128);
-  if (s == NULL ||
-      (strcmp(s, "ZlutouckyKun") &&
-       strcmp(s, "Zlutouck'yKun") &&
-       strcmp(s, "?lu?ou?k?K??"))) {
-    tvherror(LS_MAIN, "iconv() routine is not working properly (%s), aborting!", s);
-    tvh_safe_usleep(2000000);
-    abort();
-  }
-  free(s);
-}
-
 void
 intlconv_init( void )
 {
   pthread_mutex_init(&intlconv_lock, NULL);
   pthread_mutex_init(&intlconv_lock_src, NULL);
-  intlconv_test();
 }
 
 void